Windows CI sometimes fails since a few days ago

```
ddnet_engine_shared.lib(std-391022a4250a8b9a.std.feb3b897-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __imp_NtCreateFile referenced in function _ZN3std3sys7windows2fs20open_link_no_reparse17h9b3d5753800c4b4fE [D:\a\ddnet\ddnet\debug\game-client.vcxproj]
ddnet_engine_shared.lib(std-391022a4250a8b9a.std.feb3b897-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __imp_RtlNtStatusToDosError referenced in function _ZN3std3sys7windows2fs20open_link_no_reparse17h9b3d5753800c4b4fE [D:\a\ddnet\ddnet\debug\game-client.vcxproj]
ddnet_engine_shared.lib(std-391022a4250a8b9a.std.feb3b897-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __imp_NtReadFile referenced in function _ZN3std3sys7windows6handle6Handle16synchronous_read17h609a485d92162e8dE [D:\a\ddnet\ddnet\debug\game-client.vcxproj]
ddnet_engine_shared.lib(std-391022a4250a8b9a.std.feb3b897-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __imp_NtWriteFile referenced in function _ZN3std3sys7windows6handle6Handle17synchronous_write17h5e143db420a86fa8E [D:\a\ddnet\ddnet\debug\game-client.vcxproj]
D:\a\ddnet\ddnet\debug\DDNet.exe : fatal error LNK1120: 4 unresolved externals [D:\a\ddnet\ddnet\debug\game-client.vcxproj]

ddnet_engine_shared.lib(std-391022a4250a8b9a.std.feb3b897-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __imp_NtCreateFile referenced in function _ZN3std3sys7windows2fs20open_link_no_reparse17h9b3d5753800c4b4fE [D:\a\ddnet\ddnet\debug\game-server.vcxproj]
ddnet_engine_shared.lib(std-391022a4250a8b9a.std.feb3b897-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __imp_RtlNtStatusToDosError referenced in function _ZN3std3sys7windows2fs20open_link_no_reparse17h9b3d5753800c4b4fE [D:\a\ddnet\ddnet\debug\game-server.vcxproj]
ddnet_engine_shared.lib(std-391022a4250a8b9a.std.feb3b897-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __imp_NtReadFile referenced in function _ZN3std3sys7windows6handle6Handle16synchronous_read17h609a485d92162e8dE [D:\a\ddnet\ddnet\debug\game-server.vcxproj]
ddnet_engine_shared.lib(std-391022a4250a8b9a.std.feb3b897-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol __imp_NtWriteFile referenced in function _ZN3std3sys7windows6handle6Handle17synchronous_write17h5e143db420a86fa8E [D:\a\ddnet\ddnet\debug\game-server.vcxproj]
D:\a\ddnet\ddnet\debug\DDNet-Server.exe : fatal error LNK1120: 4 unresolved externals [D:\a\ddnet\ddnet\debug\game-server.vcxproj]
```

https://github.com/Robyt3/ddnet/actions/runs/5221674664/jobs/9426241593

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -737,6 +737,7 @@ if(TARGET_OS STREQUAL "windows")
   list(APPEND PLATFORM_LIBS bcrypt userenv) # for Rust (https://github.com/rust-lang/rust/issues/91974)
   list(APPEND PLATFORM_LIBS ole32) # CoInitialize(Ex)
   list(APPEND PLATFORM_LIBS shell32)
+  list(APPEND PLATFORM_LIBS ntdll) # https://github.com/ddnet/ddnet/issues/6725
 elseif(TARGET_OS STREQUAL "mac")
   find_library(CARBON Carbon)
   find_library(COCOA Cocoa)

